force-app/main/default/reports/NPSP_Outbound_Fund_Reports/Aging_Disbursements.report-meta.xml

```diff
@@ -44,7 +44,7 @@
     <columns>
         <field>Funding_Program__c.FundingRequests__r.Disbursements__r$Status__c</field>
     </columns>
-    <description>List of overdue disbursements group by funding program.</description>
+    <description>List of overdue disbursements grouped by funding program.</description>
```

Comment:
@nicolersmith - This is the description of the `Aging Report` that needs your approval.